blues/simulation.py

```diff
@@ -1002,8 +1019,8 @@ def run(self, nIter=None, nstepsNC=None, moveStep=None, nstepsMD=None, write_mov
             self._sync_states_md_to_ncmc_()
```

Comment:
Not sure I'm a huge fan of the name `sync_states_md_to_ncmc` as I spent a minute or two looking for an equivalent reverse function `sync_states_ncmc_to_md` and wondering why you weren't applying it. (The doc string doesn't help, as it says it retrieves the MD state and uses it to update the NCMC state). But after reading it, it looks more like it retrieves BOTH states. Should it just be called `sync_md_and_ncmc_states`?

Also note typo in doc string, should be "velocities" not "velocities"

Reply:
Not particularly expensive, but isn't there an extra call to `self._sync_states_md_to_ncmc_()` on line 1015? Specifically it's almost first within the `for` loop on 1016 so the call on 1015 seems redundant?